src/content/site/rules.md

```diff
@@ -123,14 +123,14 @@ Try not to, honestly, if you are looking to prevent a specific kind of attack fe
 
 # Roles
 
-TPH has a relatively small amount of roles, but each of our roles have a purpose. We don't have 30 language roles and a staff rank. That being said, it's kind of tricky to keep a track of all of the roles at once - so here you can see what they're for.
+TPH has a relatively small amount of roles, but each of our roles has a purpose. We don't have 30 language roles and a staff rank. That being said, it's kind of tricky to keep a track of all of the roles at once - so here you can see what they're for.
 
-_Community-Member, Active Contributor, Chatterbox, Super-Talky-Guy, Bepis-Drinker, Conke-Commander and Patriotic-Procrastinator_
+_Community-Member, Active Contributor, Chatterbox, Super-Talky-Guy, Bepis-Drinker, Conke-Commander, and Patriotic-Procrastinator_
 
-These are the activity roles of the server. Different tiers will grant different perks, such unlocking hidden channels, voice access, and nicknaming privileges. These roles are assigned automatically as you participate in the server, so don't ask for them.
+These are the activity roles of the server. Different tiers will grant different perks, such as unlocking hidden channels, voice access, and nicknaming privileges. These roles are assigned automatically as you participate in the server, so don't ask for them.
 
 **Wizard**<br />
-This is for helpful members who have shown exceptional aptitude or expertise in some field. This is given out on the discretion of Elliott - though other staff members may suggest someone to get it to Elliott. Less than 0.001% of users get this role.
+This is for helpful members who have shown exceptional aptitude or expertise in some field. This is given out at the discretion of Elliott - though other staff members may suggest for Elliot to give it to someone. Less than 0.001% of users get this role.
```

Comment:
As we're here, can we mention Elliot and Fox?

Reply:
What about them?

Reply:
I mean, Elliott and Fox co-own. I think this text is legacy. You editing this bit made it obvious.